### PR TITLE
Oracle Migration Guide

### LINKED ISSUE #3539: Migrate from Oracle to CockroachDB
In 2.1, we added migration guides for Postgres and MySQL. In 2.2, we should add one for Oracle, covering data import (see below), schema and application changes, and important best practices.

----

@drewdeally we should document an example of exporting data from Oracle and importing into CockroachDB.  Maybe two different techniques 1) small data set and 2) larger data set.  Lets work on the small data set example first.

### PR DESCRIPTION
**Pull request by lhirata**
Closes #3539.

### CODE REVIEW

**Comment by drewdeally** [comment]
The choice is up to user who will know best on the delim

**Comment by lhirata** [comment]
Most up-to-date link: http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/21b336e71eadc3c9ab6e507ff337c2a5926cd37d/dev/migrate-from-oracle.html

**Comment by drewdeally** [comment]
Looks good to me

http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/21b336e71eadc3c9ab6e507ff337c2a5926cd37d/dev/migrate-from-oracle.html#step-9-refactor-application-sql <http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/21b336e71eadc3c9ab6e507ff337c2a5926cd37d/dev/migrate-from-oracle.html#step-9-refactor-application-sql>

Not sure step 9 is needed.   As for docs it would need to be much more thorough and a brand new section of app refactoring with step by step example.    That was really meant to be a high level.

**Comment by lhirata** [comment]
@Amruta-Ranade / @jseldess - can you please review when you have a chance? FYI, @rolandcrosby will be helping me with the delimiter stuff.

File: v19.2/migrate-from-oracle.md [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+
+Format the generated CSV files to meet the CockroachDB's [CSV requirements](#csv-requirements).
+
+### CSV requirements
+
+You will need to export one CSV file per table, with the following requirements:
+
+- Files must be in [valid CSV format](https://tools.ietf.org/html/rfc4180), with the caveat that the delimiter must be a single character.  To use a character other than comma (such as a tab), set a custom delimiter using the [`delimiter` option](import.html#delimiter).
+- Files must be UTF-8 encoded.
+- If one of the following characters appears in a field, the field must be enclosed by double quotes:
```
Comment (Amruta-Ranade) [inline v19.2/migrate-from-oracle.md]: nit: Should "double quote" be "double-quote"? If yes, change throughout the doc.

**Comment by lhirata** [comment]
@jseldess / @rolandcrosby can you take a final look? thanks!

**Comment by lhirata** [comment]
Follow up re: data type mapping: https://github.com/cockroachdb/docs/issues/5236

**Pull request merged**